### PR TITLE
Name last missing fields in SceGxmProgram

### PR DESCRIPTION
**Pull request by bythos14**
Also tidied up get_fragment_inputs.

### CODE REVIEW

File: vita3k/gxm/include/gxm/types.h [hunk 1]
```diff
@@ -1427,9 +1427,18 @@ enum SceGxmFragmentProgramInputs : int {
...
+    SCE_GXM_PROGRAM_FLAG_PER_INSTANCE_MODE = 1 << 1,
+    SCE_GXM_PROGRAM_FLAG_DISCARD_USED = 1 << 3,
+    SCE_GXM_PROGRAM_FLAG_DEPTH_USED = 1 << 4,
+    SCE_GXM_PROGRAM_FLAG_SPRITECOORD_USED = 1 << 5,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_USED = 1 << 6,
+    SCE_GXM_PROGRAM_FLAG_FRAGCOLOR_USED = 1 << 7,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_MSAA2X = 1 << 8,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_MSAA4X = 1 << 9,
+    SCE_GXM_PROGRAM_FLAG_OUTPUT_UNDEFINED = 1 << 16,
+    SCE_GXM_PROGRAM_FLAG_INDEX_USED = 1 << 17,
```
Comment (pent0) [inline vita3k/gxm/include/gxm/types.h:1440]: Very unsure about this, need testing to see if anything break. There are games that use this, and I'm not in remember of how these values work.

Comment (bythos14) [reply vita3k/gxm/include/gxm/types.h:1440]: VitaGL's implementation of glClear still works, and it uses the INDEX semantic. So it's fine.

Comment (pent0) [reply vita3k/gxm/include/gxm/types.h:1440]: Yeah that's true. I don't think anyone is available to test further more, so I will assume it all goes right.

**Review by pent0 — APPROVED** [review]

**Comment by pent0** [comment]
I will rerun to valid format-check, after that will merge. Thanks!

**Comment by pent0** [comment]
Sorry forgot lol

**Pull request merged**